ChromeExtension/js/offscreen.js

```diff
@@ -0,0 +1,3 @@
+setInterval(async () => {
+    (await navigator.serviceWorker.ready).active.postMessage('keepAlive');
```

Comment:
https://developer.chrome.com/docs/extensions/develop/migrate/to-service-workers#keep-sw-alive